server/@types/index.d.ts

```diff
@@ -129,6 +130,14 @@ export interface EmailUploadValidatorArgs {
   actionedByUserId?: string
 }
 
+export interface NoteDocumentUploadValidatorArgs {
```

Comment:
It's ok if we need this short-term, but wouldn't the existing `EmailUploadValidatorArgs` definition work?

Reply:
-> TODO